mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -545,6 +689,34 @@ func getStateAPIKeys(d *schema.ResourceData) (newAPIKeys, changedAPIKeys, remove
 	return
 }
 
+func getStateLimits(d *schema.ResourceData) (newLimits, changedLimits, removedLimits []interface{}) {
```

Comment:
let's explain what this function is doing since the logic is not straightforward
```suggestion
// getStateLimits performs ...
func getStateLimits(d *schema.ResourceData) (newLimits, changedLimits, removedLimits []interface{}) {
```

Reply:
extracted this method to avoid duplicate code in other cases, added a comment explaining what it does.